tests/test_serialize_sequence.cc

```diff
@@ -564,12 +564,12 @@ TEST_CASE("serialize_sequence: sequence name escaping 2", "[serialize_sequence]"
     REQUIRE(after[0] == "A bell"); // This is strictly speaking not required
 
     Sequence deserialize_seq = load_sequence(temp_dir + "/" + after[0]);
-    // REQUIRE(sequence.get_label() == deserialize_seq.get_label());
+    REQUIRE(sequence.get_label() == deserialize_seq.get_label());
     // Compare all chars but not the control char which is at index 1:
     REQUIRE(sequence.get_label().substr(0,1) == deserialize_seq.get_label().substr(0,1));
     REQUIRE(sequence.get_label().substr(2) == deserialize_seq.get_label().substr(2));
-    // Control char shall be encoded as blank
-    REQUIRE(deserialize_seq.get_label().substr(1, 1) == " ");
+    // Control char shall be encoded as a control character for bell (\b)
+    REQUIRE(deserialize_seq.get_label().substr(1, 1) == "\b");
```

Comment:
Maybe we should heed the comment in line 547 and make control characters illegal for sequence labels – just call `check_for_control_characters()` in the constructor and in `set_label()`?